src/main/java/frc/robot/Robot.java

```diff
@@ -40,156 +40,51 @@
-                intakeOneBallAuto(), drive.auto(AutoPaths.twoBallLeftTwo),
-                shootOneBallAuto(), parallel("Stop Shooter", stopShooter()));
-    }
-
-    // Shoot One ball and taxi
-    public CommandBase oneBallAuto() {
-        return sequence("One Ball Auto", shootOneBallAuto(),
-                drive.auto(AutoPaths.oneBallLeftOne));
     }
 
-    private CommandBase weekTwoAutoHigh() {
-        return sequence("Week Two Auto High",
-                shootHigh(),
-
-                parallel("Stop and drive",
-                        sequence("Stop shooter", new WaitCommand(2), shooter.stop()),
-                        drive.driveDistance(2.8, 0, 0.5)),
-                parallel("Reset Arms", leftClimber.resetArms(), rightClimber.resetArms()));
-    }
-
-    private CommandBase delayedAuto() {
-        return sequence("Delayed Auto",
-                new WaitCommand(2),
-                shootHigh(),
-
-                parallel("Stop and drive",
-                        sequence("Stop shooter", new WaitCommand(2), shooter.stop()),
-                        drive.driveDistance(2.8, 0, 0.5)),
-                parallel("Reset Arms", leftClimber.resetArms(), rightClimber.resetArms()));
-    }
-
-    private CommandBase weekTwoAutoLow() {
-        return sequence("Week Two Auto Low",
-                shootLow(),
-
-                parallel("Stop and drive",
-                        sequence("Stop shooter", new WaitCommand(2), shooter.stop()),
-                        drive.driveDistance(2.8, 0, 0.5)),
-                parallel("Reset Arms", leftClimber.resetArms(), rightClimber.resetArms()));
-    }
-
-    private CommandBase onlyShoot() {
-        return sequence("Only Shoot", shootHigh());
-    }
-
-    @Autonomous
-    public CommandBase threeRightAuto = threeRightAuto();
-    @Autonomous
-    public CommandBase twoRightAuto = twoRightAuto();
-    @Autonomous
-    public CommandBase twoLeftAuto = twoLeftAuto();
+    @Autonomous(defaultAuto = true)
+    private CommandBase twoLeftAuto = auto.twoLeftAuto();
     @Autonomous
-    public CommandBase oneBallAuto = oneBallAuto();
+    private CommandBase weekTwoAutoLow = auto.weekTwoAutoLow();
     @Autonomous
-    public CommandBase weekTwoAutoLow = weekTwoAutoLow();
+    private CommandBase weekTwoAutoHigh = auto.weekTwoAutoHigh();
     @Autonomous
-    public CommandBase weekTwoAutoHigh = weekTwoAutoHigh();
+    private CommandBase onlyShoot = auto.onlyShoot();
     @Autonomous
-    public CommandBase onlyShoot = onlyShoot();
-    @Autonomous(defaultAuto = true)
-    public CommandBase delayedAuto = delayedAuto();
+    private CommandBase delayedAuto = auto.delayedAuto();
+
+    private CommandBase shootHigh() {
+        return sequence("Shoot High", shooter.setTargetAndStartShooter(HubSpeed.HIGH),
+                shooter.waitUntilSpeedUp(),
+                ballTransport.loadShooter(), ballTransport.moveBothMotorsToLaser(), new WaitCommand(0.5));
```

Comment:
Why does this have a wait at the end? If it didn't, you could combine this and `shootLow` and give it a parameter for the speed.

Reply:
It needs it so the cargo doesn't collide mid-air. The isn't a problem with shooting into the low goal